CODEOWNERS

```diff
@@ -1 +1 @@
-*       @aalavian @anantsimran @chrisochoatri @isaak-willett @marccarre
+*       @aalavian @anantsimran @chrisochoatri @convexquad @pickles-bread-and-butter @marccarre
```

Comment:
@pickles-bread-and-butter could you remove the update to this file? I posted https://github.com/woven-planet/wicker/pull/64 to ask to have me added.